app/views/roadmap/index.html.erb

```diff
@@ -9,63 +13,42 @@
       <div class="govuk-grid-row">
         <div class="govuk-grid-column-two-thirds">
           <%= render "govuk_publishing_components/components/title", title: t('roadmap.hero.heading'), margin_top: 0 %>
-          <p class="govuk-body"><%= t('roadmap.hero.description') %></p>
+          <p class="govuk-body govuk-roadmap__intro"><%= t('roadmap.hero.description') %></p>
         </div>
 
         <div class="govuk-grid-column-one-third">
           <%= render "image" %>
         </div>
       </div>
     </section>
-    <section class="govuk-roadmap-section">
+
+    <section class="govuk-roadmap-section govuk-!-padding-bottom-6">
       <div class="govuk-grid-row">
         <div class="govuk-grid-column-two-thirds">
           <%= render "govuk_publishing_components/components/heading", {
-            text: t('roadmap.numbers_section.heading_large'),
+            text: "Programme updates",
```

Comment:
Should this text live in the locale file?

Reply:
Yes. In fact it's already in there. Odd. Corrected.